rfc041-server-api-versioning.md

```diff
@@ -178,6 +184,15 @@ Vary: X-Ops-Server-API-Version`
 `$actual_version` MUST indicate the API version level of the response.  The `Vary` header is included
 in the event it becomes desirable to support caching of specific responses.
 
+#### Response Headers
+
+Any request to the server will return the min and max supported API versions as headers in the following format:
```

Comment:
You should explain that this is necessary to detect older servers that don't support versioning.